npm/pkg/controlplane/translation/translatePolicy_test.go

```diff
@@ -1571,8 +1667,12 @@ func TestIngressPolicy(t *testing.T) {
 			npmNetPol.PodSelectorIPSets, npmNetPol.PodSelectorList, err = podSelectorWithNS(npmNetPol.NameSpace, policies.EitherMatch, tt.targetSelector)
```

Comment:
can we have UTs for the errors from podSelectorWithNS if we don't already have them?